contracts/MarginCalculator.sol

```diff
@@ -72,13 +73,22 @@ contract MarginCalculator is Initializable {
         // Vault contains no short tokens: return collateral value.
         if (_isEmptyAssetArray(_vault.shortOtokens)) return (SignedConverter.intToUint(collateralAmount.value), true);
 
+        // get required margin, denominated in strike or underlying asset
```

Comment:
this is a little confusing that for puts it is denominated in strike and call in underlying but the same function returns both.